.github/ISSUE_TEMPLATE/issue-template.md

```diff
@@ -1,16 +1,29 @@
 ---
-name: Issue template
-about: Standard issue template.
-title: "[Replace me with meaningful title]"
+name: Peerlist Request
+about: Request to be added to the allow list for the 3Box Labs hosted anchor service
+title: "[Replace me with Project Name]"
 labels: ''
 assignees: ''
-
 ---
+# Team
+
+Team name or your github handle if you are a team of one
+
+# Use Case
+
+A few words about what your node will be used for so we can make recommendations for your setup
+
+# Overview
+How are you running your nodes? What cloud infrastructure? How are you running your IPFS node(s)?
 
-# Description
+## Multiaddress persistence
```

Comment:
I think this one could maybe be removed as with go-ipfs now it's okay if your multiaddr changes?  I'll leave it to @v-stickykeys to confirm though